shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -4,19 +4,52 @@ import logger from '../../../../logger'
 import * as Constants from '../../../../constants/chat'
 import * as Types from '../../../../constants/types/chat'
 import * as ChatGen from '../../../../actions/chat-gen'
-import {Notifications, type Props} from '.'
-import {connect, type TypedState} from '../../../../util/container'
+import {Notifications} from '.'
+import {compose, connect, type TypedState, lifecycle, type Dispatch} from '../../../../util/container'
 import {type DeviceType} from '../../../../constants/types/devices'
 
+type StateProps = {
+  channelWide: boolean,
+  desktop: Types.NotifyType,
+  mobile: Types.NotifyType,
+  muted: boolean,
+  saveState: Types.NotificationSaveState,
+}
+
 type DispatchProps = {
-  _resetSaveState: (conversationIDKey: Types.ConversationIDKey) => void,
-  _onMuteConversation: (conversationIDKey: Types.ConversationIDKey, muted: boolean) => void,
+  _resetSaveState: (conversationIDKey: Types.ConversationIDKey) => any,
```

Comment:
i made these return any as i added the type Dispatch from util/container which types dispatch correctly. we commonly return the value of dispatch which actually isn't void